.changeset/config.json

```diff
@@ -1,12 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.6.3/schema.json",
-  "changelog": [
```

Comment:
Why are you removing this, this is repo config

Reply:
Actually, this was not on purpose. Must have occurred during the upstream merge

Reply:
I'll revert this